test/uk/gov/hmrc/mtdsatestsupportapi/domain/StatusEnumSpec.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.mtdsatestsupportapi.domain
+
+import support.UnitSpec
+import uk.gov.hmrc.mtdsatestsupportapi.models.domain.StatusEnum
+import utils.enums.EnumJsonSpecSupport
+
+class StatusEnumSpec extends UnitSpec with EnumJsonSpecSupport {
```

Comment:
As we're adding the custom writes method it is worth having tests that check that when we write the model to json that it does it in it's `downstreamValue` format (we still also want a test for the reads as well though but that will be straightforward)